### ISSUE TITLE
New compiler warning since release of FirebaseFirestore 0.10.0

### ISSUE THREAD

**Issue by ghowen** [firebase/FirebaseUI-iOS#394]
```
ld: warning: instance method 'isEqual:' in category from /Users/georg/Projekte/Apps/mypois/Pods/FirebaseUI/FirebaseUIFrameworks/FirebaseFirestoreUI/Frameworks/FirebaseFirestoreUI.framework/FirebaseFirestoreUI(FUIBatchedArray.o) overrides method from class in /Users/georg/Projekte/Apps/mypois/Pods/FirebaseFirestore/Frameworks/FirebaseFirestore.framework/FirebaseFirestore(FIRDocumentSnapshot_ba50e18b2764d8ebf57764b4c76ad173.o)
ld: warning: instance method 'hash' in category from /Users/georg/Projekte/Apps/mypois/Pods/FirebaseUI/FirebaseUIFrameworks/FirebaseFirestoreUI/Frameworks/FirebaseFirestoreUI.framework/FirebaseFirestoreUI(FUIBatchedArray.o) overrides method from class in /Users/georg/Projekte/Apps/mypois/Pods/FirebaseFirestore/Frameworks/FirebaseFirestore.framework/FirebaseFirestore(FIRDocumentSnapshot_ba50e18b2764d8ebf57764b4c76ad173.o)
```

My guess is that FirebaseFirestore now implements isEqual and hash in DocumentSnapshots.

**Comment by morganchen12** [comment]
Fixed in 8240720536b3895fdf3b72e117e8932b0173f38d, I'll close this bug when it's released.

Thanks for reporting!

**Comment by morganchen12** [comment]
The fix has been released in v4.5.2.
